mapie/calibrators/standard.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from typing import List, cast
+
+import numpy as np
+from sklearn.utils.validation import _num_samples
+
+from mapie._typing import ArrayLike, NDArray
+from mapie.calibrators.base import BaseCalibrator
+from .ccp.utils import check_required_arguments
+from mapie.conformity_scores import ConformityScore
+
+
+class StandardCalibrator(BaseCalibrator):
+    """
+    Calibrator used to get the standard conformal prediciton. It is strictly
+    equivalent to ``MapieRegressor`` with ``method='base'``.
+
+    Attributes
+    ----------
+    fit_attributes: Optional[List[str]]
+        Name of attributes set during the ``fit`` method, and required to call
+        ``predict``.
+
+    q_up_: float
+        Calibration fitting results, used to build the upper bound of the
+        prediction intervals. It correspond to the quantile of the calibration
+        conformity scores.
+
+    q_low_: float
+        Same as q_up_, but for the lower bound
+    """
+    fit_attributes: List[str] = ["q_up_", "q_low_"]
+
+    def __init__(self) -> None:
+        return
+
+    def fit(
+        self,
+        X_calib: ArrayLike,
```

Comment:
if you give X_calib is it truely equivalent to MapieRegressor ?

Reply:
I wanted to do the splitting in the main class (``SplitCPRegressor``) and not in the calibrators, to have the maximum in the main class and the simplest calibrator possible. However, so calibrators may need the training or calibration data (ex: CQR, would need both). So here, I specify X_calib, but no worries, it is indeed equivalent to ``MapieRegressor``.